03week/towersOfHanoi.js

```diff
@@ -13,29 +13,67 @@ let stacks = {
   c: []
 };
 
+const stackA=stacks.a;
+const stackB=stacks.b;
+const stackC=stacks.c;
+const stacksArr=[stackA, stackB, stackC];
+// Create an array that houses all stacks so a forEach function can be applied(shortens code).
+
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
 function movePiece() {
-  // Your code here
+
+     let startStack.pop();
```

Comment:
this would throw an error, you need to assign the variable correctly, in addition you should use const. You also need to pass the start and end stacks to this function
```javascript
const piece = startStack.pop();
endStack.push(piece);
```